README.md

```diff
@@ -49,7 +49,7 @@ cd bank-of-anthos
 
 ```
```

Comment:
Add a line in this step to define the zone, similar to `PROJECT_ID=<your-project-id>` above

```
ZONE=<your-zone>
```

Reply:
Pushed. Please take a look when ever you get a mo. Have a great weekend!

Reply:
Actually, I think this might work better here under step 3 **Create a Kubernetes cluster**.  That is:

```
ZONE=<your-zone>
gcloud beta container clusters create bank-of-anthos \
    --project=${PROJECT_ID} --zone=us-central1-b \
    --project=${PROJECT_ID} --zone=${ZONE} \
    --machine-type=n1-standard-2 --num-nodes=4
```

Reply:
Yes, I agree. That flows better...pushed.